### DISCUSSION TITLE
Power output FAST Farm

### DISCUSSION THREAD

**Discussion by windexplorer21** [OpenFAST/openfast#2282] [Q&A]
Dear all,

I am simulating a very simple wind farm consisting of two turbines (IEA 15MW) under steady wind conditions (10.69 m/s rated speed), **without shear or turbulence intensity**. I am curious about how the software evaluates the power output of the **second** turbine, as I am encountering a discrepancy when compared to the power curve of the turbine obtained from an OpenFAST simulation.

Specifically, I have recorded a power output of 5320 kW for the second turbine. However, the wind speed necessary to achieve this power is significantly higher compared to the wind speed that the second turbine is actually experiencing, which is 6.95 m/s (this value is the average from 9 points across the rotor).

![Power](https://github.com/OpenFAST/openfast/assets/167337679/e09b5b2b-e187-4dcf-bc34-f0708f86bafa)

Thank you in advance for your help.

Best regards,

windexplorer21

**Comment by jjonkman** [comment]
Dear @windexplorer21,

FAST.Farm will use the wake from the upstream turbine combined with the inflow as the inflow that the downstream turbine will experience.  If the average of your 9 inflow points does not correspond to the wind speed you expect is necessary for the downstream turbine to generate the power it is generating, I would question whether your 9 inflow points are adequately representative of the inflow the downstream turbine is experiencing.  You could compare your 9-point average to AeroDyn output `RtVavgxh`, which is axial component of the rotor-disk averaged relative wind velocity as tracked by AeroDyn.

Best regards,

**Reply by windexplorer21** [reply]
Dear @jjonkman, 

thank you for your suggestion. I placed the 9 points like this:
![Averaging points](https://github.com/OpenFAST/openfast/assets/167337679/18cb0ee9-2052-4814-9c0d-b402763f7863)

I obtain 7.123 m/s from `RtVavgxh`. The wind speed that I should have to obtain a power output of 5320kW is 7.415 m/s. So there is still a small difference. 

windexplorer21

**Reply by jjonkman** [reply]
Dear @windexplorer21,

There is likely a variation of wind speed across the rotor disk and the wind speed at some areas of the disk may have a greater impact on power capture than at other areas.  So, I'm not too surprised that there is a small difference between the velocities.

Best regards,

**Reply by windexplorer21** [reply]
Dear @jjonkman,

Thank you, I appreciate you taking the time to provide me this explanation.

Best regards,
@windexplorer21

**Reply by windexplorer21** [reply]
Dear @jjonkman,

I apologize for bothering you again, but I have encountered another issue related to the question I asked you previously. I am simulating the IEA-15-240-RWT-Monopile turbines in the farm and I am obtaining different Ct curves from an OpenFAST simulation compared to those provided in the deafult excel file regarding this turbine. I think this difference may affect my results.
[IEA-15-240-RWT_tabular.xlsx](https://github.com/user-attachments/files/16027821/IEA-15-240-RWT_tabular.xlsx)
![Ct Comparison](https://github.com/OpenFAST/openfast/assets/167337679/215ad6f1-0b58-4f0c-8956-80429fffeafc)
What do you think about this? 
I apologize once more for the disturbance. Thank you.

Kind regards,

windexplorer21

**Reply by jjonkman** [reply]
Dear @windexplorer21,

I'm not familiar with the spreadsheet you linked to, but I would guess that there are differences in the model setup, e.g., aerodynamic model selection or rigid versus flexible (or perhaps the version of the model).

Best regards,
